pkg/services/sqlstore/apikey.go

```diff
@@ -38,6 +40,20 @@ func (ss *SQLStore) GetAPIKeys(ctx context.Context, query *models.GetApiKeysQuer
 	})
 }
 
+// GetAPIKeys queries the database based
+// on input on GetApiKeysQuery
+func (ss *SQLStore) GetBasicAPIKeys(ctx context.Context) []*models.ApiKey {
```

Comment:
```suggestion
func (ss *SQLStore) GetNonServiceAccountAPIKeys(ctx context.Context) []*models.ApiKey {
```

nit: Not sure if we should call It basic, this is going to be the new "standard" of apikeys. Is there a less generic name for these?